import XCTest

final class A2Tests: XCTestCase {
    func testSuccessA2() throws {}

    func testFailureA2() throws {
        XCTFail("Failure at A2")
    }
}
